src/components/Dashboard.tsx

```
@@ -63,29 +59,30 @@ export default function Dashboard() {
-      if (!monitoredDates.some((d) => d.getTime() === date.getTime())) {
-        setMonitoredDates([...monitoredDates, date]);
-      }
     }
   };
 
   const handleNotificationSave = async (
     type: "email" | "webhook",
     value: string
   ) => {
-    if (session) {
+    if (session && session.user) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)